core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -231,15 +231,15 @@ private RequestMetricSupport() {}
             failure = parent.counter(requests, idPrefix.tags("result", "failure"));
 
             requestDuration = newTimer(
-                    parent, idPrefix.name("requestDuration"), idPrefix.tags());
+                    parent, idPrefix.name("request.duration"), idPrefix.tags());
             requestLength = newDistributionSummary(
-                    parent, idPrefix.name("requestLength"), idPrefix.tags());
+                    parent, idPrefix.name("request.size"), idPrefix.tags());
             responseDuration = newTimer(
-                    parent, idPrefix.name("responseDuration"), idPrefix.tags());
+                    parent, idPrefix.name("response.duration"), idPrefix.tags());
             responseLength = newDistributionSummary(
-                    parent, idPrefix.name("responseLength"), idPrefix.tags());
+                    parent, idPrefix.name("response.size"), idPrefix.tags());
```

Comment:
`response.length` vs `response.size` - we discussed this internally in our team and this was the suggested approach. If you feel the `response.length` form is better, please let me know; I am willing to change this part of the PR if required.

Reply:
I'm not sure which is better to be honest. Perhaps we need to do some quick research on what others use.

Reply:
Uh, never mind. I realized you already reverted back to `length`.